index.js

```diff
@@ -11,10 +11,28 @@ function CompressionPlugin(options) {
 	this.asset = options.asset || "{file}.gz";
 	this.algorithm = options.algorithm || "gzip";
 	if(typeof this.algorithm === "string") {
-		var zlib = require("zlib");
-		this.algorithm = zlib[this.algorithm];
-		if(!this.algorithm) throw new Error("Algorithm not found in zlib");
-		this.algorithm = this.algorithm.bind(zlib);
+		if (this.algorithm === "zopfli") {
+			try {
+				var zopfli = require("node-zopfli");
+			} catch(err) {
+				throw new Error("node-zopfli not found");
+			}
+			this.algorithm = function (content, fn) {
+				zopfli.gzip(content, {
+					verbose: options.verbose ? options.verbose : false,
+					verbose_more: options.verbose ? options.verbose : false,
+					numiterations: options.verbose ? options.verbose : 15,
+					blocksplitting: options.verbose ? options.verbose : true,
```

Comment:
There's no way to set `options.blocksplitting` to `false` as the ternary would then evaluate the right side.

```js
options.hasOwnProperty('blocksplitting') ? options.blocksplitting : true,
```